importer/importer.go

```diff
@@ -137,15 +140,22 @@ func (im *Importer) createTable(server servers.ClickhouseServer, user apikeys.AP
 		storagePolicy = server.GetStoragePolicy()
 	}
 
+	engine := "MergeTree"
+	clusterStmt := ""
+	if cluster := user.GetDBCluster(); cluster != "" {
+		engine = "ReplicatedMergeTree('/clickhouse/{cluster}/tables/{shard}/{database}/{table}', '{replica}')"
```

Comment:
@poundifdef Following [this discussion](https://github.com/scratchdata/ScratchDB/pull/44#discussion_r1385041101), as we have  default path set in
https://github.com/scratchdata/ScratchDB/blob/dba1aea19116998b7976bbd533771c0868efd7c1/docker/clickhouse/server/config.d/cluster.xml#L2-L3

it is possible to update this to:


```suggestion
		engine = "ReplicatedMergeTree"
```